pkg/registry/apis/playlist/register.go

```diff
@@ -57,7 +57,24 @@ func (b *PlaylistAPIBuilder) InstallSchema(scheme *runtime.Scheme) error {
 		&playlist.Playlist{},
 		&playlist.PlaylistList{},
 	)
-	return nil
+
+	// Link this version to the internal representation.
+	// This is used for server-side-apply (PATCH), and avoids the error:
+	//   "no kind is registered for the type"
```

Comment:
adding the error that is logged when server side apply fails you would find it here